recipes/espresso/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+./src/Parasail/build
+
 mkdir -p $PREFIX/bin
 cp src/ESPRESSO_C.pl $PREFIX/bin/
 cp src/ESPRESSO_Q.pl $PREFIX/bin/
 cp src/ESPRESSO_Q_Thread.pm $PREFIX/bin/
 cp src/ESPRESSO_S.pl $PREFIX/bin/
 cp src/ESPRESSO_Version.pm $PREFIX/bin/
+cp src/Parasail.pm $PREFIX/bin/
+cp src/Parasail.so $PREFIX/bin/
```

Comment:
Should this file be in bin/ or lib/?

Reply:
Parasail.so is for this perl module: https://github.com/Xinglab/espresso/blob/v1.5.0/src/Parasail/lib/Parasail.pm
I tried a local build with Parasail.so in lib/ and the test for `use Parasail` failed. Putting Parasail.so in bin/ with Parasail.pm works. Probably there is a better way to package this, but my perl knowledge is limited